content/docs/using-regolith/configuration.md

```diff
@@ -188,7 +181,7 @@ i3-wm.mod: Mod1
 i3-wm.alt: Mod4
 ```
```

Comment:
I'm not sure why github is highlighting these :point_up: in red... they are still there according to my local copy.

Reply:
I think there is just some markdown parsing rule that's being tripped unintentionally.  I don't think it matters.

Reply:
I wish GitHub hadn't chosen the same color as a "deletion" to highlight the parsing issue :laughing: